seq2science/rules/configuration_logging.smk

```diff
@@ -10,10 +13,35 @@ onstart:
             src = os.path.join(os.getcwd(), file)
             dst = os.path.join(config['log_dir'], os.path.basename(file) if n<2 else "profile.yaml")
             shutil.copy(src, dst)
+
 onsuccess:
     if config.get("email") not in ["none@provided.com", "yourmail@here.com", None]:
         os.system(f"""echo "Succesful pipeline run! :)" | mail -s "The seq2science pipeline finished succesfully." {config["email"]} 2> /dev/null""")
+
 onerror:
+    logger.info(
+"""
+
+
+  __    __    __  ____  ____  _   
+ /  \  /  \  /  \(  _ \/ ___)/ \  
+(  O )(  O )(  O )) __/\___ \\\\_/  
+ \__/  \__/  \__/(__)  (____/(_)  
+
+
+One or more rules did not finish as expected!
+
+
+Please take a look at our Frequently Asked Questions for help: 
+https://vanheeringen-lab.github.io/seq2science/content/faq.html
```

Comment:
I was thinking something like

> One or more rules did not finish as expected!
> 
> 
> Please take a look at the log files of the failed rule(s), and our Frequently Asked Questions: 
https://vanheeringen-lab.github.io/seq2science/content/faq.html

to save users the extra click. Didn't see the faq describes this immediately, so we could skip it :)